tests/include.bats

```diff
@@ -47,3 +47,33 @@ load test_helper
   assert_success
   assert_output "DONE"
 }
+
+@test "includes a file only once" {
```

Comment:
In this test you're only testing that `BASHER_INCLUDED_ONCE` is correctly set. The test shouldn't actually care about the internals of how `include` keeps its state. A better approach would be to use a script which outputs something and check that the output is only printed once, even if the file is included two times.

Reply:
Nevermind, I saw the actual test now. We should probably just delete the check on `BASHER_INCLUDED_ONCE`